diff -urN exim_surbl-2.3.org/config.ini exim_surbl-2.3/config.ini
--- exim_surbl-2.3.org/config.ini	1970-01-01 01:00:00.000000000 +0100
+++ exim_surbl-2.3/config.ini	2019-05-13 13:26:40.581571134 +0200
@@ -0,0 +1,5 @@
+[config]
+max_file_size = 50000
+surbl_enable = 1
+uribl_enable = 1
+dbl_enable = 1
diff -urN exim_surbl-2.3.org/exim_surbl.pl exim_surbl-2.3/exim_surbl.pl
--- exim_surbl-2.3.org/exim_surbl.pl	2019-05-13 13:28:21.937931916 +0200
+++ exim_surbl-2.3/exim_surbl.pl	2019-05-13 13:28:43.838583229 +0200
@@ -20,6 +20,8 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+use Config::IniFiles;
+
 sub surblspamcheck
 {
 
@@ -29,6 +31,8 @@
 # Please see the following website for details on usage of
 # this script:  http://www.teuton.org/~ejm/exim_surbl
 
+    my $cfg = new Config::IniFiles -file => '/etc/exim-surbl/config.ini';
+
     # The following variable is the full path to the file containing the 
     # two-level top level domains (TLD).
     # ---------------------------------------------------------------------
@@ -56,13 +60,13 @@
     # This variable defines the maximum MIME file size that will be checked
     # if this script is called by the MIME ACL.  This is primarily to
     # keep the load down on the server.  Size is in bytes.
-    my $max_file_size = 50000;
+    my $max_file_size = $cfg->val('config', 'max_file_size', 50000);
     
     # The following variables enable or disable the SURBL, URIBL and DBL
     # lookups.  Set to 1 to enable and 0 to disable.
-    my $surbl_enable = 1;
-    my $uribl_enable = 0;
-    my $dbl_enable = 1;
+    my $surbl_enable = $cfg->val('config', 'surbl_enable', 1);
+    my $uribl_enable = $cfg->val('config', 'uribl_enable', 0);
+    my $dbl_enable = $cfg->val('config', 'dbl_enable', 1);
     
     # Check to see if a decode MIME attachment is being checked or 
     # just a plain old text message with no attachments
