diff -urN exim_surbl-2.3.org/exim_surbl.pl exim_surbl-2.3/exim_surbl.pl
--- exim_surbl-2.3.org/exim_surbl.pl	2019-06-26 12:39:29.690473235 +0200
+++ exim_surbl-2.3/exim_surbl.pl	2019-06-26 12:40:08.281619931 +0200
@@ -20,6 +20,8 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+use Net::DNS::Resolver;
+
 sub surblspamcheck
 {
 
@@ -91,7 +93,17 @@
         # the return message for the SURBL lookup.
         my @params = @_;
         my $surbldomain = ".multi.surbl.org";
-        @dnsbladdr=gethostbyname($params[0].$surbldomain);
+	my $res = Net::DNS::Resolver->new(
+		nameservers => ['8.8.8.8', '8.8.4.4', '1.1.1.1',  '1.0.0.1'],
+	);
+	my $reply = $res->search(@params[0].$surbldomain);
+	if ($reply) {
+		foreach my $rr ($reply->answer) {
+			next unless $rr->type eq "A";
+			print $rr->address, "\n";
+			@dnsbladdr = $rr->address;
+		}
+	}
         # If gethostbyname() returned anything, build a return message.
         $return_string = "";
         if (scalar(@dnsbladdr) != 0) {
